File: js/challenge0016/challenge0016.c
#include <stdio.h>
#include <stdbool.h>

bool lessThanOrEqualToZero(const int x) {
    return x <= 0;
}

int main(){
    
    printf("%d\n", lessThanOrEqualToZero(5));   // 0
    printf("%d\n", lessThanOrEqualToZero(0));   // 1
    printf("%d\n", lessThanOrEqualToZero(-2));  // 1
}

File: js/challenge0016/challenge0016.pl
sub lessThanOrEqualToZero{
	$_[0] <= 0 ? "true" : "false";
}

print lessThanOrEqualToZero(5);
print "\n";
print lessThanOrEqualToZero(0);
print "\n";
print lessThanOrEqualToZero(-2);
